css/styles.css

```diff
@@ -79,3 +73,52 @@ label {
   display: flex;
   justify-content: right;
 }
+
+ul {
+  padding: 0;
+  list-style-type: none;
+}
+
+p {
+  line-height: 1.5em;
+  font-weight: normal;
+}
+
+li {
+  margin: 10px 0;
+}
+
+@media screen and (min-width: 768px) {
+  img {
+    height: 350px;
+  }
+
+  h2 {
+    margin-top: 0;
+  }
+
+  li {
+    margin: 0 0 50px;
+  }
+}
```

Comment:
Media queries should always be the last thing in your stylesheet, since source order is a major factor in your media query working as intended.